File: sdkwork-app-sdk-csharp/Models/ActivityRecordVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class ActivityRecordVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? Id { get; set; }
        public string? ActivityId { get; set; }
        public string? ActivityTitle { get; set; }
        public string? ActivityType { get; set; }
        public string? ParticipateTime { get; set; }
        public string? Status { get; set; }
        public string? Result { get; set; }
        public Dictionary<string, object>? Rewards { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/activity_record_vo.go
package types

// 活动参与记录
type ActivityRecordVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id string `json:"id"`
	ActivityId string `json:"activityId"`
	ActivityTitle string `json:"activityTitle"`
	ActivityType string `json:"activityType"`
	ParticipateTime string `json:"participateTime"`
	Status string `json:"status"`
	Result string `json:"result"`
	Rewards map[string]interface{} `json:"rewards"`
}
